### ISSUE TITLE
[BUG] Preferences not being saved

### ISSUE THREAD

**Issue by iamdannywyatt** [aonez/Keka#999]
- Keka version: 1.2.50 (4808)
- macOS version: 10.15.7

When I go to the preferences > Finder Extension, and add Full Mac to the list of locations where the contextual menu should show up, after a while I go back and it's not there anymore.
This is what I set:
![Keka's bug](https://user-images.githubusercontent.com/96640778/147376438-c1a2619e-dda1-40e8-9310-06623ee55af3.png)

And then after a while, it goes back to the default:
![image](https://user-images.githubusercontent.com/96640778/147376445-b92a67ea-c434-47ff-a262-4b3e4860c6ad.png)

**Comment by aonez** [comment]
I was unable to reproduce this one. Maybe the preferences got reseted anyhow? Maybe you're using an un-sandboxed Keka and a sandboxed one? They do not share the preferences.

**Comment by iamdannywyatt** [comment]
It’s not something that happens once I close and open the app. It’s pretty random. 
I’m not using more than one instance of Keka. I only have version 1.2.50 (4808) installed.
I have another volume with Keka installed, but that’s not where I load the app from.
I will see if that happens again, now that I’ve delete that other volume’s app and all files related to the app.
If that happens again, I will let you know.
Thanks

**Comment by roeleveld** [comment]
I have the same issue.
OS 13.2
Keka 1.2.6
Keka works fine, it just doesn't save any of my preferences (e.g. deleting zips after decompressing = true).

**Comment by aonez** [comment]
@roeleveld be sure you set that preferences in the Keka Preferences. The settings in the advanced windows (the little one) are only for the current session and not permanent.

![Screenshot 2023-02-08 at 19 34 02](https://user-images.githubusercontent.com/480163/217620891-22520672-4bd5-4a92-9765-c7b3305ee91a.png)

**Comment by roeleveld** [comment]
Yep. I also confirmed this on a vanilla user account, just to rule out any local issues.
